README.md

```diff
@@ -21,7 +21,11 @@ Chain adapter uses `chain-adapter.properties` as a default configuration file th
 - `chain-adapter.iroha` - Iroha host and port configuration
 - `chain-adapter.dropLastReadBlock` - as it was mentioned before, chain adapter saves the last read block height. It's possible to drop height to zero on a service startup by setting this value to `true`. Good for testing purposes. 
 - `chain-adapter.irohaCredential` - credentials of the account that will be used by the service to listen to Iroha blocks. The account must have `can_get_blocks` permission.
+- `chain-adapter.clusterEnabled` - `true` if expected to run multiple instances of the service and `false` otherwise. If `true` Iroha blocks will be deduplicated. Default value is `false`.
+- `chain-adapter.clusterHazelcastMembers` - comma separated list of Hazelcast's node addresses in format `host1:port1,host2:port2`. Required only if `chain-adapter.clusterEnabled` is `true`.
 
+=false
```

Comment:
Is it a mistake with formatting or something else?

Reply:
Yes, this is mistake. Deleted it. Thank you